store/tikv/rawkv_test.go

```diff
@@ -126,6 +132,33 @@ func (s *testRawKVSuite) TestSimple(c *C) {
 	c.Assert(err, NotNil)
 }
 
+func (s *testRawKVSuite) TestBatchPut(c *C) {
+	testNum := 0
+	testKeys := make([][]byte, 0)
```

Comment:
`var testKeys [][]byte` is more commonly useds. See https://github.com/golang/go/wiki/CodeReviewComments#declaring-empty-slices